redis.tf

```diff
@@ -0,0 +1,24 @@
+module "pcs_redis" {
+  source                        = "git@github.com:hmcts/cnp-module-redis?ref=master"
+  product                       = var.product
+  location                      = azurerm_resource_group.rg.location
+  env                           = var.env
+  common_tags                   = var.common_tags
+  redis_version                 = "6"
+  business_area                 = "sds"
```

Comment:
```suggestion
  business_area                 = "cft"
```

Reply:
resolved the feedback to now use cft instead of sds.